auth/build.gradle.kts

```diff
@@ -33,14 +33,9 @@ dependencies {
     api(Config.Libs.PlayServices.auth)
 
     compileOnly(Config.Libs.Provider.facebook)
-    // Needed to override Facebook
-    implementation(Config.Libs.Support.cardView)
-    implementation(Config.Libs.Support.customTabs)
+    implementation(Config.Libs.Support.cardView) // Needed to override Facebook
```

Comment:
Did something change about how Facebook uses custom tabs?

Reply:
Nope, but we use custom tabs ourselves (for the ToS and PP) so it was a duplicate dependency:
https://github.com/firebase/FirebaseUI-Android/blob/df554e2f8cec7ae594ea5d05e5ce61e39281f4d2/auth/build.gradle.kts#L25